src/components/Contentful/Hero.js

```diff
@@ -47,7 +48,7 @@ export default function ContentfulHero({
         <div className="row">
           <div className="col-7">
             {sectionNameComponent}
-            <h1 className="contentful-hero__page-title">{pageTitle}</h1>
+            <h1 className="contentful-hero__page-title" dangerouslySetInnerHTML={{__html: threeSpaceToLineBreak(pageTitle)}} />
```

Comment:
You should be able to define this function in the Hero.js file:

```js
function threeSpaceToLineBreak(pageTitle) {
  return pageTitle.split('   ').splice(1, 0, <br />)
}
```

Then use it like so:

```jsx
<h1 className="contentful-hero__page-title">{threeSpaceToLineBreak(pageTitle)}</h1>
```